packages/playwright-core/src/server/network.ts

```diff
@@ -430,11 +430,12 @@ export class Response extends SdkObject {
   }
 
   httpVersion(): string {
-    if (!this._httpVersion)
+    const httpVersion = this._httpVersion?.toUpperCase();
```

Comment:
Let's be careful and only convert 'h2/h2c' into 'HTTP/2.0'. There is stuff like spdy or quic.